signalfx-org-metrics/metrics.yaml

```diff
@@ -1,143 +1,267 @@
-    the ingestion volume limit is reached.
+sf.org.apm.numAddSpansCallsByToken:
+  brief: The number of calls to the `/v2/trace` endpoint for a specific access token.
+  description: 'The number of calls to the `/v2/trace` endpoint for a specific access token.
 
-    Dimension(s): `orgId`, `tokenId`
+  Dimension(s): `orgId`, `tokenId`
 
-    Data resolution: 1 second'
+  Data resolution: 1 second'
   metric_type: counter
-  title: sf.org.apm.numSpansDroppedThrottleByToken
+  title: sf.org.apm.numAddSpansCallsByToken
 
-sf.org.apm.grossSpansReceived:
-  brief: The number of spans SignalFx receives before filtering and throttling.
-  description: 'The number of spans SignalFx receives before filtering or throttling.
+sf.org.apm.numContainers:
+  brief: The number of containers actively sending data to Splunk APM.
+  description: 'The number of containers actively sending data to Splunk APM.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 minute'
+  metric_type: gauge
+  title: sf.org.apm.numContainers
+
+sf.org.apm.numHosts:
+  brief: The number of hosts that are actively sending data to Splunk APM.
+  description: 'The number of hosts that are actively sending data to Splunk APM.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 minute'
+  metric_type: gauge
+  title: sf.org.apm.numHosts
+
+sf.org.apm.grossSpanBytesReceived:
+  brief: The number of bytes Splunk APM receives from spans after decompression
+    but before filtering and throttling.
+  description: 'The number of bytes Splunk APM receives from spans after
+    decompression but before filtering and throttling.
 
   Dimension(s): `orgId`
 
   Data resolution: 1 second'
   metric_type: counter
-  title: sf.org.apm.grossSpansReceived
+  title: sf.org.apm.grossSpanBytesReceived
 
-sf.org.apm.grossSpansReceivedByToken:
-  brief: The number of spans SignalFx receives for a specific access token
-    before filtering and throttling.
-  description: 'The number of spans SignalFx receives for a specific access token
-    before filtering or throttling.
+sf.org.apm.numSpanBytesReceived:
+  brief: The number of bytes Splunk APM accepts for a span after decompression
+    after filtering and throttling.
+  description: 'The number of bytes Splunk APM accepts for a span after
+    decompression after filtering and throttling.
 
-    Dimension(s): `orgId`, `tokenId`
+    Dimension(s): `orgId`
 
     Data resolution: 1 second'
   metric_type: counter
-  title: sf.org.apm.grossSpansReceivedByToken
+  title: sf.org.apm.numSpanBytesReceived
 
-sf.org.apm.numSpansReceived:
-  brief: The number of spans SignalFx accepts after filtering and throttling.
-  description: 'The number of spans SignalFx accepts after filtering and throttling.
+sf.org.apm.grossSpanBytesReceivedByToken:
+  brief: The number of bytes Splunk APM receives for a specific access token
+    from spans after decompression but before filtering and throttling.
```

Comment:
`from spans` is also a good expression to use also with `sf.org.apm.numSpanBytesReceived`